src/include/columnar/cstore_version_compat.h

```diff
@@ -36,16 +36,11 @@
 #define TTS_EMPTY(slot) ((slot)->tts_isempty)
 #define ExecForceStoreHeapTuple(tuple, slot, shouldFree) \
 	ExecStoreTuple(newTuple, tupleSlot, InvalidBuffer, shouldFree);
+#define table_open heap_open
```

Comment:
Minor: in engine this is `#define table_open(r, l) heap_open(r, l)` I guess both would work as expected as long as there is no variable `table_open` in the code.